MotionMark/resources/strings.js

```diff
@@ -60,6 +60,7 @@ var Strings = {
         scoreUpperBound: "scoreUpperBound",
         fps: "fps",
         bootstrap: "bootstrap",
+        bootstrapIterations: "bootstrapIterations",
```

Comment:
This is side table really useful? It seems like we can just hard-code these strings where it's used.

Reply:
I don't really know why this strings table was used. I agree it's unwieldy.